bin/sozo/Cargo.toml

```diff
@@ -32,6 +32,7 @@ dojo-world.workspace = true
 hex = "0.4.3"
 hex-literal = "0.4.1"
 itertools.workspace = true
+jsonrpsee = { version = "0.16.2", default-features = false }
```

Comment:
i dont see this being used anywhere?

Reply:
Forgot to remove, thanks for the good catch. Was when trying version matching for reqwest.

Reply:
removed.